pkg/provider/aws/resources/vpc.go

```diff
@@ -139,6 +148,17 @@ func (vpc *Vpc) GetPrivateSubnets(dag *core.ResourceGraph) []*Subnet {
 	return subnets
 }
 
+func (vpc *Vpc) GetSecurityGroups(dag *core.ResourceGraph) []*SecurityGroup {
```

Comment:
At this point, why not have the SecurityGroups as a field on VPC?

Reply:
because then dependencies reflect would go the incorrect way in the dependencies it initializes

Reply:
Ah, gotcha - the SGs are created in the VPC